src/main/java/lotto/service/LottoService.java

```diff
@@ -0,0 +1,124 @@
+package lotto.service;
+import lotto.domain.AutoLotto;
+import lotto.domain.Lotto;
+import lotto.domain.WinningLotto;
+import lotto.domain.rule.LottoRules;
+import lotto.domain.rule.ResultCalculateRules;
+import lotto.domain.rule.WinningRules;
+import java.util.stream.Collectors;
+import java.util.*;
+
+import static lotto.domain.message.LottoErrorMessage.WINNING_NUMBER_FORMAT_ERROR;
+import static lotto.domain.rule.LottoRules.AUTO_LOTTO_PRICE;
+import static lotto.domain.message.LottoPriceErrorMessage.INVALID_LOTTO_PRICE_DIVISIBLE_OR_ZERO;
+import static lotto.utils.DefaultErrorMessage.INVALID_INTEGER_FORMAT;
+import static lotto.utils.DefaultErrorMessage.NULL_OR_EMPTY_INPUT;
+
+public class LottoService {
+
+    public List<AutoLotto> createAutoLottosByLottoPrice(String inputTotalLottoPrice) {
+        List<AutoLotto> autoLottos = new ArrayList<>();
+
+        validateDefaultInputPrice(inputTotalLottoPrice);
+        validateLottoPrice(Integer.parseInt(inputTotalLottoPrice));
+        int autoLottoCount = Integer.parseInt(inputTotalLottoPrice) / AUTO_LOTTO_PRICE.getValue();
+        for (int i = 0; i < autoLottoCount; i++) {
+            autoLottos.add(Lotto.createRandomLotto());
+        }
+        return autoLottos;
+    }
+
+    public WinningLotto createWinningLotto(String inputWinningLottoNumbers) {
+        try {
+            List<Integer> winningNumbers = Arrays.stream(inputWinningLottoNumbers.split(","))
+                    .map(String::trim)
+                    .map(Integer::parseInt)
+                    .toList();
+
+            Lotto winningLotto = new Lotto(winningNumbers); // 로또 공통 유효성 검증
+            return Lotto.createWinningLotto(winningLotto.getNumbers());
+        } catch (Exception e) {
+            throw new IllegalArgumentException(WINNING_NUMBER_FORMAT_ERROR.getMessage());
+        }
+    }
+
+    public WinningLotto setWinningLottoBonusNumber(WinningLotto winningLotto, String bonusNumber) {
+        winningLotto.setBonusNumber(bonusNumber);
+        return winningLotto;
+    }
+
+
+    public Map<WinningRules, Long> calculateResults(List<AutoLotto> autoLottos, WinningLotto winningLotto) {
+        return autoLottos.stream()
+                .collect(Collectors.groupingBy(
+                        autoLotto -> determineWinningRule(autoLotto, winningLotto),
+                        () -> new EnumMap<>(WinningRules.class),
+                        Collectors.counting()
+                ));
+    }
+
+    public float calculateWinningStatistics(Map<WinningRules, Long> results, List<AutoLotto> autoLottos) {
+        long totalPrize = ResultCalculateRules.ZERO_LOTTO_PRICE.getValue().longValue();
+        for (WinningRules rank : WinningRules.values()) {
+            if (rank != WinningRules.NO_MATCH) {
+                totalPrize += rank.getPrize() * results.getOrDefault(rank, ResultCalculateRules.DEFAULT_COUNT.getValue().longValue());
```

Comment:
1. result라는 변수명이 모호한 것 같습니다.
2. `getOrDefault` 부분에서 찾고자 하는 랭크가 존재하지 않으면 DEFAULT_COUNT의 값을 불러왔는데, 해당하지 않은 랭크가 없다면 0개라는건 기정사실이 아닌가 싶습니다. 해당 값이 추후에 변경될 가능성이 매우 적을것 같은데 상수화를 한 이유는 무엇인가요?

```suggestion
        for (WinningRules rank : WinningRules.values()) {
            if (rank != WinningRules.NO_MATCH) {
                totalPrize += rank.getPrize() * results.getOrDefault(rank, 0L);
```